docs/installation/keycloak/keycloak-minikube.md

```diff
@@ -0,0 +1,178 @@
+kubectl config use-context keycloak-oidc
+```
+
+5. Test the Configuration: To verify that the configuration is working, list the namespaces in the cluster by executing the command:
+
+```
+kubectl get ns
+```
+
+Upon running this command, a new browser window will open, prompting you to log in. Once you've completed the login process, you can close the window. You should see the namespaces in your cluster.
+
+Note: Make sure to replace `<YOUR-KEYCLOAK-URL>`, `<REALM-NAME>`, `<CLIENT-ID>`, and `<CLIENT-SECRET>` with your specific Keycloak configuration details wherever necessary.
+
+
+# Setting up Headlamp with OIDC Configuration
+
+To configure Headlamp, you can use the Headlamp Helm chart. Follow these steps to set it up with OIDC (OpenID Connect) authentication:
+
+1. Before setting up Headlamp Go to Keycloak Admin Dashboard, click on Client > Client Details and Edit Client, Add `http://localhost:4466/oidc-callback` to the list of `Valid redirect URIs`.
+
+![Keycloak redirect URL](./keycloak-redirect.jpeg)
+
+2. Create a `values.yaml` file and add the following OIDC configuration to it:
+
+   ```yaml
+   config:
+     oidc:
+       clientID: "<YOUR-CLIENT-ID>"
+       clientSecret: "<YOUR-CLIENT-SECRET>"
+       issuerURL: "<YOUR-KEYCLOAK-URL>/realms/<REALM-NAME>"
+       scopes: "email,profile"
+    ```
+
+Replace `<YOUR-CLIENT-ID>`, `<YOUR-CLIENT-SECRET>`, `<YOUR-KEYCLOAK-URL>`, and `<REALM-NAME>` with your specific OIDC configuration details.
+
+3. Save the `values.yaml` file and Install Headlamp using helm with the following commands:
+
+```
+helm repo add headlamp https://headlamp-k8s.github.io/headlamp/
+helm install headlamp-oidc headlamp/headlamp -f values.yaml --namespace=headlamp --create-namespace
+```
+
+![Headlamp install](./headlamp-install.jpg)
+
+This will install Headlamp in the headlamp namespace with the OIDC configuration from the values.yaml file.
+
+
+4. After a successful installation, you can access Headlamp by port-forwarding to the pod:
+
+```
+kubectl port-forward svc/headlamp-oidc 4466:80 -n headlamp
+```
+
+5. Open your web browser and go to http://localhost:4466. Click on "sign-in." After completing the login flow successfully, you'll gain access to your Kubernetes cluster using Headlamp.
+
+![Headlamp access](./headlamp-access1.jpg)
+![Headlamp access](./headlamp-access2.jpg)
+![Headlamp access](./headlamp-access3.jpg)
+
+## Conclusion
+
+In this tutorial, we've explored the process of setting up Headlamp within a Kubernetes cluster, integrating it with OIDC (OpenID Connect) authentication provided by Keycloak. By following the steps outlined in this guide, you've successfully configured Headlamp to enhance your Kubernetes cluster management.
+
+This setup allows you to benefit from Headlamp's user-friendly interface and advanced features, all while ensuring secure and streamlined authentication through Keycloak. With the power of OIDC, you can easily and safely access and manage your Kubernetes resources.
```

Comment:
small change, missing an 'a'
`all while ensuring secure and streamlined authentication through Keycloak.`
to
`all while ensuring a secure and streamlined authentication through Keycloak.`